plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/__init__.py

```diff
@@ -12,11 +12,16 @@
 from omegaconf import MISSING
 
 
+@dataclass
+class RayConf:
+    init: Dict[str, Any] = field(default_factory=lambda: {"address": None})
```

Comment:
I know we have been through it but asking one last time:
can this dict be a Structured Config?

```
@dataclass
class RayInitConf:
  address: Optional[str] = None
```

Reply:
let me address this as a follow up (ray init takes in a bunch of params)
I want to get this in soon so we can get the post unblocked :)